helix-view/src/handlers/dap.rs

```diff
@@ -20,11 +21,11 @@ macro_rules! debugger {
 
 // general utils:
 pub fn dap_pos_to_pos(doc: &helix_core::Rope, line: usize, column: usize) -> Option<usize> {
-    // 1-indexing to 0 indexing
-    let line = doc.try_line_to_char(line - 1).ok()?;
-    let pos = line + column.saturating_sub(1);
-    // TODO: this is probably utf-16 offsets
-    Some(pos)
+    lsp_pos_to_pos(
+        doc,
+        Position::new(line.saturating_sub(1) as u32, column as u32),
+        OffsetEncoding::Utf16,
+    )
 }
```

Comment:
The change to `dap_pos_to_pos` is still here. Like I said it's the right change but better split into a seperate PR, potentially together with the type changes to `u32` (as it's an orthogonal change).

Reply:
oh yeah, sorry, forgot to commit it after I removed it :sweat: